kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/KubernetesAssistantDefaultResourceLocator.java

```diff
@@ -0,0 +1,37 @@
+package org.arquillian.cube.kubernetes.impl;
+
+import java.net.URL;
+import java.util.Optional;
+
+public class KubernetesAssistantDefaultResourceLocator {
+
+    private static final String[] RESOURCE_NAMES =
+        new String[] {"openshift", "META-INF/fabric8/openshift", "kubernetes", "META-INF/fabric8/kubernetes"};
```

Comment:
I think that if you are using Kubernetes assistant we should not find for OpenShift resources